app/views/katello/api/v2/capsule_content/sync_status.json.rabl

```diff
@@ -2,6 +2,15 @@ object @capsule
 
 attribute :last_sync_time
 
+node :types_to_skip do
```

Comment:
Something about the word 'skip' isn't really jiving with me. It almost gives the impression that this was somehow configured or determined by the user but that doesn't seem to be. Is  `unsyncable_content_types` more clear? This is nitpicking but it's worth being as deliberate as possible. Let me know if you really like the current naming too.

Reply:
`unsyncable_content_types` works, sure.